### PR TITLE
Add remove_value_output attribute to onnx ArgMax extractor

### PR DESCRIPTION
**Pull request by yekruglov**
Root cause analysis: ArgMax is converted to TopK with 2 Result nodes. One of them should be removed from the IR (extensions/back/TopKNormalizer.py), but it does not happen, because `remove_value_output` attribute is missing.  

Solution: Add `remove_value_output` to onnx ArgMax extractor, like it is done in tf and caffe extractors

Ticket: CVS-50941

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names

Validation:
* [x]  Unit tests
* [x]  Framework operation tests
* [x]  Transformation tests
* [x]  Model Optimizer IR Reader check

Documentation:
* [x]  Supported frameworks operations list N/A: no new operations were enabled
* [x]  Guide on how to convert the **public** model N/A: no new models were enabled
* [x]  User guide update N/A

### CODE REVIEW

**Comment by yekruglov** [comment]
@achetver Please, check IR Reader part

**Comment by achetver** [comment]
Solution looks consistent with changes in MO and was checked offline. :+1:

File: model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class TopKExtender(Extender):
 
     @staticmethod
     def extend(op: Node):
-        if op.graph.graph['cmd_params'].framework in ('tf', 'caffe'):
+        if op.graph.graph['cmd_params'].framework in ('tf', 'caffe', 'onnx'):
```
Comment (achetver) [inline model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py:27]: Please, update copyright year in this file.

Comment (yekruglov) [reply model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py:27]: Done

**Review by achetver — APPROVED** [review]

**Review by vgavrilo — APPROVED** [review]

**Review by rkazants — APPROVED** [review]

**Review by iimironov — APPROVED** [review]

**Comment by yekruglov** [comment]
@lazarevevgeny PR is ready for merge

File: model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py (hunk 1 above)
Comment (lazarevevgeny) [inline model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py:27]: This is not correct. What if the original ONNX model has operation "TopK"? In this case we should keep both outputs! The same comment for TF model.

The Extender logic to check whether the output with values should be removed or not based on connection of the outputs.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py (hunk 1 above)
Comment (yekruglov) [reply model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py]: Ok, I changed this. @achetver Can you check this condition? It should be consistent with code in TopKNormalizer.

Comment (achetver) [reply model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py]: Yes, checked, in works correct for models with TopK operation with both one and two outputs.

**Review by achetver — APPROVED** [review]

File: model-optimizer/extensions/front/onnx/argmax_ext.py [hunk 2]
```diff
@@ -35,7 +35,9 @@ def extract(cls, node):
             'out_max_val' : 0,
 
             # Set attribute to trigger ArgMax replacer in case do not keep the dimension
-            'keepdims': keepdims
```
Comment (lazarevevgeny) [inline model-optimizer/extensions/front/onnx/argmax_ext.py:38]: Please, create a ticket to implement extractor for ONNX operation ArgMin.

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**